client/src/SignupNonProfit.tsx

```diff
@@ -59,129 +101,184 @@ function SignupNonProfit() {
+                                <Grid item md={6} xs={12}>
+                                    <TextField
+                                        id="city"
+                                        label="City"
+                                        placeholder="City"
+                                        value={formData.city}
+                                        onChange={handleFieldChange}
+                                    />
+                                </Grid>
+                                <Grid item md={6} xs={12}>
+                                    <TextField
+                                        id="state"
+                                        label="State"
+                                        placeholder="State"
+                                        value={formData.state}
+                                        onChange={handleFieldChange}
+                                    />
+                                </Grid>
+                                <Grid item md={6} xs={12}>
+                                    <TextField
+                                        id="ein"
+                                        label="Entity Identification Number (EIN)"
+                                        placeholder="EIN"
+                                        value={formData.ein}
+                                        onChange={handleFieldChange}
+                                    />
+                                </Grid>
+                                <Grid item md={6} xs={12}>
+                                    <TextField
+                                        id="tax_exempt_id"
+                                        label="Tax Exempt ID"
+                                        placeholder="Tax Exempt ID"
+                                        value={formData.tax_exempt_id}
+                                        onChange={handleFieldChange}
+                                    />
+                                </Grid>
+                                <Grid item md={8} xs={12}>
+                                    <Select
+                                        id="nonprofit_classification"
+                                        label="IRS Nonprofit Organization Classification"
+                                        placeholder="Select classification"
+                                        options={exemptions}
+                                        value={formData.nonprofit_classification}
+                                        onChange={handleSelectChange}
+                                    />
+                                </Grid>
+                                    <Grid item md={8} xs={12}>
+                                            <Button 
+                                                onClick={handleNextClick}
+                                                className={classes.button}
+                                                fullWidth
```

Comment:
on this and the submit button, let's pass a `disabled` prop so that user can't move on to next section of the form until they've completed all the inputs
https://v4.mui.com/api/button/#props

Reply:
Disabled prop? No problem! Figuring out the simplest way to validate the form is taking a bit of research!

Reply:
Most of what i found involved external libraries and your solve at the bottom was super clever! Will show both there (see below)